lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -49,17 +51,19 @@ public static function assertValidIdentifier($identifier)
     /**
      * {@inheritDoc}
      */
-    public function getSubstringExpression($value, $position, $length = null)
+    public function getSubstringExpression($value, $from, $length = null)
```

Comment:
Let's keep the argument names consistent with `master`:
https://github.com/doctrine/dbal/blob/27285b19369ccb1a40df66bcf91d535a739b746b/lib/Doctrine/DBAL/Platforms/AbstractPlatform.php#L738

When reworking the signatures, I tried to use names from the SQL analogs of the platform methods.